### PR TITLE
More small fixes

### PR DESCRIPTION
**Pull request by jherico**
* [Allow user browser to work with marketplace](https://app.asana.com/0/27650181942747/83158649553224)
* [Better preference dialog initial state](https://app.asana.com/0/27650181942747/83769475464641)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: d7d65e3b5551b0c2926979e1a564ac278ae2a446 available at: https://gist.github.com/416b32e5704256460779<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2c2c1e832928702f379e6cdb4e668e76ca675a05 available at: https://gist.github.com/416b32e5704256460779<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.dmg

**Comment by howard-stearns** [comment]
When I try the PR build, and create an in-world browser and go to marketplace, I still get the json file display instead of the download.

Steps:
0. Install pr build on Windows. Run it, confirming that it displays (build PR6953), and that the changes are there (e.g., that c:/Program Fiels/High Fidelity - PR6953/resources/qml/Browser.qml has the new code).
1. Use edit.js to make an in-world browser.
2. click on login and enter credentials.
3 click on marketplace and pick, e.g., the crate "get" button. Get json in browser rather than the crate.

File: interface/resources/qml/dialogs/PreferencesDialog.qml [hunk 1]
```diff
@@ -62,6 +62,13 @@ Window {
 
                 sections.push(sectionBuilder.createObject(prefControls, { name: category }));
             }
+
+            if (sections.length) {
+                sections[0].expanded = true;
+                if (sections.length === 1) {
+                    sections[0].expandable = false
```
Comment (howard-stearns) [inline interface/resources/qml/dialogs/PreferencesDialog.qml]: Maybe "collapsable" would be a better name for this property?
It _seems_ weird to set expanded to true, and then expandable to false. 

(The logic isn't weird, given the actual meaning, but it's jarring to read. I wouldn't bother if there are no other changes for this PR, but given that the web browser part of the PR doesn't seem to be working, I figured its worth making the suggestion.)

Comment (jherico) [inline interface/resources/qml/dialogs/PreferencesDialog.qml]: I was thinking about that as I was writing it, but I didn't go back and change it.  It's a good idea and I'll update the PR.

**Comment by cozza13** [comment]
I tested this and the functionality works

**Comment by howard-stearns** [comment]
I just spoke with Chris. He was testing from marketplace button -- which I think always worked, with or without this pr. I was testing from an in-world browser per my repro steps. He's going to recheck.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 18d6a09dc9c3cf20ed5f15390bc336ed9aafc1be available at: https://gist.github.com/416b32e5704256460779<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.dmg

**Comment by jherico** [comment]
@howard-stearns ok, I misunderstood your browser thing.  The bug I fixed was specifically for the browser window launched with Ctrl-B, not for web entities inside the world.  The latter would be significantly harder and isn't necessarily a use case we want to support.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 04e6a5360a03d3ede5dbd39d32bae109839a315a available at: https://gist.github.com/416b32e5704256460779<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6953.dmg

**Comment by howard-stearns** [comment]
Got it.
I've updated the ticket to reflect that, and tested the new (rebased?) build with this in mind.
Works.

**Pull request merged**